### DISCUSSION TITLE
Can we approve a minor update for Mitt?

### DISCUSSION THREAD

**Discussion by Luciden** [developit/mitt#177] [General]
Great job with Mitt. We really enjoy using it. Our customers won't let us use a component that hasn't been updated in a year for fear that it's no being maintained. Do you think we can do just a minor readme update or something simple to get Mitt 3.0.1?

If the license were open we could fork it and maintain ourselves.

**Comment by joeycumines** [comment]
The project is under the MIT license? There's not a CLA and there's only one name in the LICENSE file, but it's a tiny open-source project, and it's something of a social faux pas to add your name as a contributor.

Just fork it?

**Reply by Luciden** [reply]
I agree that would be most logical but we have to get approval to take over maintenance of it which is also difficult. I know the regulations applied to us don't make a whole lot of sense but we don't have a choice.
